libraries/AP_Rally/AP_Rally.cpp

```diff
@@ -86,6 +94,11 @@ bool AP_Rally::set_rally_point_with_index(uint8_t i, const RallyLocation &rallyL
 
     _last_change_time_ms = AP_HAL::millis();
 
+    DataFlash_Class *df = DataFlash_Class::instance();
```

Comment:
Nitpick: Whitespace, this is aligned with the line above but not the rest of the function/rules. (It's possible I counted spaces wrong on the web interface and the other lines are wrong and this is right, but I think I counted correctly...)